MetaMask/metamask-extension
# fix: Notifications platform links

## **Description**

This PR introduces the logic to handle links within notifications to support the different platforms on which we can use the extension.

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/24708?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Open the notifications list page
2. Click on a product announcement
3. Check the link in the detail page

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
